testkit/src/main/scala/org/bitcoins/testkit/core/gen/CryptoGenerators.scala

```diff
@@ -109,6 +109,13 @@ sealed abstract class CryptoGenerators {
 
   def privateKey: Gen[ECPrivateKey] = Gen.delay(ECPrivateKey())
 
+  /** Generates a random non-zero private key */
```

Comment:
The comment here doesn't match the code, does it? From reading the code it looks like we're generating a privkey where none of the bytes are zero, not the privkey itself

Reply:
The private key is technically a number (mod the order of the secp256k1 group), so I don't think it is too confusing, the bytes are just the serialization and its most efficient to check that rather than doing something like `NumberUtil.toUnsignedInt(privKey.bytes) != 0`